network-api/networkapi/mozfest/blocks/carousel_text_block.py

```diff
@@ -0,0 +1,21 @@
+from wagtail import blocks
+
+from networkapi.wagtailpages.pagemodels import customblocks
+
+
+class CarouselTextBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(help_text="Heading for the block.", required=False)
+
+    text = blocks.RichTextBlock(features=["bold", "italic", "link"])
+    carousel_images = blocks.ListBlock(customblocks.ImageBlock(), max_num=4)
+
+    # Use specific link fields for the CTA on the block as opposed to the
+    # common.link_blocks.LabelledExternalLinkBlock so it can be marked as
+    # required=False.
+    link_url = blocks.URLBlock(help_text="A CTA URL for a link displayed", required=False)
```

Comment:
There is no need to internal links / page chooser?

Reply:
That's not a requirement at the moment, but we'll see with QA